packages/rapier/src/lib/components/Colliders/Collider.svelte

```diff
@@ -88,17 +95,16 @@
     if (hasRigidBodyParent) {
       const rigidBodyWorldPos = new Vector3()
       const rigidBodyWorldQuatInversed = new Quaternion()
-      object.traverseAncestors((child: Object3D) => {
-        if (child.userData.isRigidBody) {
-          child.getWorldPosition(rigidBodyWorldPos)
-          child.getWorldQuaternion(rigidBodyWorldQuatInversed)
-          rigidBodyWorldQuatInversed.invert()
-        }
-      })
+  
+      parentRigidBodyObject?.getWorldPosition(rigidBodyWorldPos)
```

Comment:
Instead of traversing all ancestors in the scene, context provided by an ancestor rigidbody is used